js/plugins/api.js

```diff
@@ -181,7 +181,8 @@ export class Api {
             credentials: 'include',
             headers: Object.assign({}, DEFAULT_HEADERS, WRITE_HEADERS, headers || {}),
         })
-        .catch(this.onError);
+        .catch(this.onError)
```

Comment:
Not exactly related to that PR but I just realized that we [don't handle HTTP errors properly](https://github.com/github/fetch#handling-http-error-statuses) in that plugin. `fetch` requires that you deal with that explicitly and doesn't consider a high status code as an error :(